File: APi/DTO/ShiftDTO .cs

namespace DTO;

public class ShiftDTO
{
    public DateTime StartTime { get; set; }
    public DateTime EndTime { get; set; }
    public bool Monday { get; set; }
    public bool Tuesday { get; set; }
    public bool Wednesday { get; set; }
    public bool Thursday { get; set; }
    public bool Friday { get; set; }
    public bool Saturday { get; set; }
    public bool Sunday { get; set; }

    public int IdMedico { get; set; }
}

File: Front/src/app/shiftDto.ts
export class shiftsDto {
    Idshift = 0;
    ShiftName: string = '';
    startDate = new Date();
    EndDate: Date | null = null;
    Sunday: boolean = true;
    Monday: boolean = true;
    Tuesday: boolean = true;
    Wednesday: boolean = true;
    Thursday: boolean = true;
    Friday: boolean = true;
    Saturday: boolean = true;
}